proof.ml

```diff
@@ -709,14 +720,15 @@ let rec rec_commute_down_weakenings proof =
                 new_head_wk_tail_wk_head_tail head tail head_wk tail_wk (Whynot e) 2 in
             let new_proof = get_commuted_proof (rec_commute_down_weakenings (Contraction_proof (new_head, e, new_tail, p))) in
             true, Weakening_proof (new_head_wk, formula, new_tail_wk, new_proof)
-    | Exchange_proof (s, permutation, Weakening_proof (head_wk, formula, tail_wk, p)) ->
+    | Exchange_proof (s, permutation1, permutation2, Weakening_proof (head_wk, formula, tail_wk, p)) ->
```

Comment:
I wonder if it would be easier for future devs to precise which perm it is. For example:
`display_permutation, permutation`

Reply:
No problem, mostly a matter of presentation. Some comments could help as well.

Reply:
Ok let me rename it, just where there are both permutations.